src/executor/upload_executor.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import base64
+
+from src.planner.upload_plan import UploadPlan
+from src.executor.abstract_executor import AbstractExecutor
+
+
+class UploadExecutor(AbstractExecutor):
+
+    def __init__(self, node: UploadPlan):
+        super().__init__(node)
+
+    def validate(self):
+        pass
+
+    def exec(self):
+        """
+        Upload the video blob into the location defined by 'path'
+        on the server. The video blob is in base64 format, so
+        it will first be decoded by the executor and then saved
+        at the specified path, prefixed by '/tmp'
+        """
+
+        video_blob = self.node.video_blob
+        path = self.node.file_path
+        video_bytes = base64.b64decode(video_blob[1:])
```

Comment:
A classification question: what is video_blob[0]?

Reply:
video_blob[0] is the byte literal prefix 'b'.